docs/application/native/guides/personal/account.md

```diff
@@ -292,13 +300,35 @@ To add your application on the Add account screen of the device:
 
    The required information includes:
 
-   - `appid="app-id name"`Application ID (for example, `appid="org.tizen.account"`).
-   - `providerid="url style string"`Provider identification string (for example, `providerid="http://www.tizen.org"`).
-   - `multiple-accounts-support="true or false"`Multiple account support (if your application can manage 2 or more accounts, set to `true`; otherwise set to `false`)
-   - `<icon section="account">application icon name</icon>`Icon displayed on the Add account screen.The icon size must be 72 x 72 px and the icon file must be in the `{app-root}\shared\res\` directory.
-   - `<icon section="account-small">application small icon name</icon>`Small icon displayed on the Other application screen to represent the account provider.The small icon file must be in the `{app-root}\shared\res\` directory.
-   - `<label xml:lang="en-gb">Application name</label>`Application name displayed on the Add account screen, according to the selected language.An account provider must have at least one label tag.
-   - `<capability>http://tizen.org/account/capability/contact</capability>`For more information, see the [Account provider properties table](#provider).
+   - `appid="app-id name"`
+
+     Application ID (for example, `appid="org.tizen.account"`).
+
+   - `providerid="url style string"`
+
+     Provider identification string (for example, `providerid="http://www.tizen.org"`).
+
+   - `multiple-accounts-support="true or false"`
+
+     Multiple account support (if your application can manage 2 or more accounts, set to `true`; otherwise set to `false`)
+
+   - `<icon section="account">application icon name</icon>`
+
+     Icon displayed on the Add account screen.The icon size must be 72 x 72 px and the icon file must be in the `{app-root}\shared\res\` directory.
```

Comment:
line 317: The separated lines are merged as one line.
line 321: The separated lines are merged as one line.
line 861: The separated lines are merged as one line, Code tag was not applied. (Table: Account properties)
line 894: The separated lines are merged as one line, Code tag was not applied. (Table: Account provider properties)

Reply:
As I left a comment o issue #265, don't need to code tag in line 861 and 894.

*=> I think String and Integer don't need to code tag because they are not code but data type.*